x/examples/fyne-tools/README.md

```diff
@@ -14,6 +16,31 @@ go run fyne.io/fyne/v2/cmd/fyne package -os android && adb install Net_Tools.apk
 Note: the generated APK is around 85MB.
 
 
+## Windows
+
+From macOS, you can build the app for Windows with MinGW.
+
+First install MinGW:
```

Comment:
AFAIK, [Mingw-w64](https://www.mingw-w64.org/) is a fork of the original mingw (a.k.a., mingw32) and is maintained by different devs.

```suggestion
From macOS, you can build the app for Windows with Mingw-w64.

First install Mingw-w64:
```

Reply:
Updated.